### ISSUE TITLE
[Feature request] Create widget(s)

### ISSUE THREAD

**Issue by wikijm** [Oliv4945/jarvis-android-app#8]
Hi @Oliv4945,

I was wondering if you plan to create some widget to start voice recognition without starting the app.

For example, [Domo Widget](https://play.google.com/store/apps/details?id=illimiteremi.domowidget) let user start voice recognition and get voice+text result outside of an app:
![domomowidget-conversation](https://cloud.githubusercontent.com/assets/10724809/25179079/acaba424-2508-11e7-8e11-9b2422aae0f4.jpg)

**Comment by Oliv4945** [comment]
Good idea @wikijm, I might do that in the future. It is added to long term todo list
By the way the option "Start app with voice recognition" is very close as you have only one icon to tap, but at the end you have the application opened.
